tests/integration/humilis-elasticache.yaml

```diff
@@ -26,10 +26,49 @@ humilis-elasticache:
                     to_port: 6379
                     source_security_group_id: {"Ref": "HumilisElasticache"}
 
-        - layer: elasticache
+        - layer: elasticache1
```

Comment:
We test the three cases, it is a bit long (as Elasticache clusters take time to be created) but it avoids surprise when we'll be using the layer.